### PR TITLE
Add docs for lazy loading

### PR DESCRIPTION
**Pull request by bergice**
Resolves https://github.com/silverstripe/silverstripe-assets/issues/450

### CODE REVIEW

File: docs/en/02_Developer_Guides/14_Files/02_Images.md [hunk 1]
```diff
@@ -200,6 +200,14 @@ SilverStripe\Core\Injector\Injector:
       Quality: 90
 ```
 
+### Lazy Loading
+
+Images will be [lazy loaded](https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading) by default.
+
+To disable lazy loading for an individual image, you can call `$MyImage.LazyLoad(false)`.
+
+You can opt out of lazy loading globally by setting the `Image::lazy_loading_enabled` config to `false`.
```
Comment (emteknetnz) [inline docs/en/02_Developer_Guides/14_Files/02_Images.md:209]: Provide yml that devs can copy paste in to their project as a code block

File: docs/en/02_Developer_Guides/14_Files/02_Images.md [hunk 2]
```diff
@@ -200,6 +200,14 @@ SilverStripe\Core\Injector\Injector:
       Quality: 90
 ```
 
+### Lazy Loading
+
+Images will be [lazy loaded](https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading) by default.
+
+To disable lazy loading for an individual image, you can call `$MyImage.LazyLoad(false)`.
```
Comment (emteknetnz) [inline docs/en/02_Developer_Guides/14_Files/02_Images.md:207]: Specify that this is only for templates

File: docs/en/02_Developer_Guides/14_Files/02_Images.md [hunk 3]
```diff
@@ -200,6 +200,14 @@ SilverStripe\Core\Injector\Injector:
       Quality: 90
 ```
 
+### Lazy Loading
+
+Images will be [lazy loaded](https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading) by default.
```
Comment (emteknetnz) [inline docs/en/02_Developer_Guides/14_Files/02_Images.md:205]:
```suggestion
Images added via the WYSIWYG editor and as objects on templates are [lazy loaded](https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading) by default.
```

**Review by emteknetnz — CHANGES_REQUESTED** [review]

File: docs/en/02_Developer_Guides/14_Files/02_Images.md (hunk 2 above)
Comment (bergice) [reply docs/en/02_Developer_Guides/14_Files/02_Images.md:207]: What do you mean?

You could technically call this via PHP as well.
Rendering-wise it currently only works with templates, but that's kind of obvious?

**Comment by bergice** [comment]
@emteknetnz feedback addressed.

File: docs/en/02_Developer_Guides/14_Files/02_Images.md (hunk 2 above)
Comment (emteknetnz) [reply docs/en/02_Developer_Guides/14_Files/02_Images.md]: You could make a quick mention that images inserted via the WYSIWYG are handled differently i.e. you cannot disable lazyloading on them by calling LazyLoad(false)

I know it's seems kind of obviousus, though we have plenty of context on this whole thing, whereas other developers reading these docs probably will not

**Comment by emteknetnz** [comment]
Failing unit test is unrelated

**Pull request merged**